float64  x
float64  y
float64  z
---
gpd/GraspConfigList grasps
int64 num_grasps